src/store/modules/app/state.js

```diff
@@ -66,7 +66,7 @@ const state = {
     { value: 'pl-PL', label: 'Polski' },
     { value: 'ms', label: 'Bahasa Malaysia' }
   ],
-  snowflakes: true
+  snowflakes: false
```

Comment:
why keeping this value in the state if the rest of the app don't have reference to it?

Reply:
To re-use it next year, maybe? That's what I would do were this my feature.

Reply:
It's still referenced globally on App.vue. this just makes it hidden and inaccessible